lib/openshift/volume_snapshot.rb

```diff
@@ -31,5 +33,30 @@ def ready?(user:, quiet: false, cached: false)
 
       return res
     end
+
+    def pvc_name(user: nil, cached: true, quiet: false)
+      rr = raw_resource(user: user, cached: cached, quiet: quiet)
+      rr.dig('spec', 'source', 'persistentVolumeClaimName')
+    end
+
+    def volumesnapshotclass_name(user: nil, cached: true, quiet: false)
+      rr = raw_resource(user: user, cached: cached, quiet: quiet)
+      rr.dig('spec', 'volumeSnapshotClassName')
```

Comment:
According to `volumeSnapshotClassName`, the name should be `volume_snapshot_class_name`